pkg/apis/extensions/v1beta1/types.go

```diff
@@ -407,6 +407,10 @@ type JobSpec struct {
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/jobs.md
 	Completions *int32 `json:"completions,omitempty"`
 
+	// Optional duration in seconds relative to the StartTime that the job may be active
```

Comment:
s/StartTime/startTime/

API comments should refer to json field names, not Go field names.